packages/rum-core/test/performance-monitoring/breakdown.spec.js

```diff
@@ -64,7 +64,7 @@ describe('Breakdown metrics', () => {
     const pageLoadTransaction = createTransaction(PAGE_LOAD)
     pageLoadTransaction.end()
     const breakdown = captureBreakdown(pageLoadTransaction, TIMING_LEVEL1_ENTRY)
-    const spanTypes = ['Network', 'DOM Processing', 'Page Render']
+    const spanTypes = ['Request', 'Response', 'Processing', 'Load']
```

Comment:
This should include `DNS`, `TCP` as well.

Reply:
The example data has 0 value for them, hence we ignore 0 duration from posting, we dont see them here.

Reply:
Decided to add an navigatio entry to have all the entry types!